### PR TITLE
feat: Stats API

### PR DESCRIPTION
**Pull request by sindharta**

### CODE REVIEW

File: Runtime/Scripts/RTCPeerConnection.cs [hunk 1]
```diff
@@ -330,6 +341,18 @@ static void OnFailureSetSessionDesc(IntPtr ptr)
                 connection.m_opSetDesc.Done();
             }, null);
         }
+
+        [AOT.MonoPInvokeCallback(typeof(DelegateSetSDFailure))]
```
Comment (karasusan) [inline Runtime/Scripts/RTCPeerConnection.cs]: This should be used value `DelegateCollectStats` not `DelegateSetSDFailure`

File: WebRTC~/Packages/com.unity.webrtc/Tests [hunk 2]
```diff
@@ -0,0 +1 @@
+../../../Tests
```
Comment (karasusan) [inline WebRTC~/Packages/com.unity.webrtc/Tests]: I am not sure this file is needed

Comment (sindharta) [reply WebRTC~/Packages/com.unity.webrtc/Tests]: I think we need this in order to have the tests inside the test project.
There is actually a better way to do this though, and I will fix this soon

File: Runtime/Scripts/RTCPeerConnection.cs (hunk 1 above)
Comment (sindharta) [reply Runtime/Scripts/RTCPeerConnection.cs]: Ok

**Comment by karasusan** [comment]
LGTM!

**Pull request merged**